### DISCUSSION TITLE
Licensing

### DISCUSSION THREAD

**Discussion by Lilith-NightRose** [Thorium-Sim/thorium-narrative#1] [General]
So, as you may have noticed, I set up the License for the narrative. For now it's CC-BY, which I think is probably the closest to Apache. I do tend to prefer copyleft licenses like CC-BY-SA, but I think it'd get lost in a quagmire w/r/t issues of RPer's emergent narrative, so I'm sticking with the more permissive option. 

If folks more familiar than I will the specifics of copyright law when it comes to roleplaying want to jump in, I'm all ears. Otherwise, let's stick to CC-BY for now.

**Comment by alexanderson1993** [comment]
I think you chose well.

I personally don't like copyleft licenses, mainly because they're more bluster than substance. The licenses only really matter if someone has the desire and means to sue over infringement, and we don't necessarily have either of those.
